src/formatDate.js

```diff
@@ -22,6 +23,10 @@ function formatDate (date,
   const hours = Math.floor(duration.asHours())
   const days = Math.floor(duration.asDays())
 
+  const formatOutput = (label, value) => {
+    return condensed === true ? `${value}${label.charAt(0)}` : `${value} ${label} ago`
```

Comment:
Can we get rid of the `charAt(0)` here? Should be able to pass in `daysLabel: 'd', condensed: true` to get what you want. Since we allow the user to specify those labels, I see no value in making `condensed: true` enforce a "first char only" limitation. 

I.e. `{ daysLabel: 'd', condensed: true }` vs `{ daysLabel: 'daysNoneOfThisMatterCuzWeOnlyUseTheFirstChart', condensed: true }`